frontend/view-edit-lead/lead-home/lead-section.component.tsx

```diff
@@ -43,6 +56,12 @@ const css = `
     font-size: 2.1rem;
   }
 
+  & .lead-metadata {
+    font-style: italic;
+    font-size: 1.2rem;
+    text-align: center;
+  }
+
```

Comment:
Same styles as the `audit trail` https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/frontend/view-edit-client/client-home/client-section.component.tsx#L86